data/migrations/20190211224425_notes.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('notes', table => {
+    table.increments();
+
+    table.string('title', 255).notNullable();
+    table.string('content', 10000).notNullable();
```

Comment:
You could also do something like this
```suggestion
    table.text('content').notNullable();
```